tools/vale/styles/config/vocabularies/edgeless/accept.txt

```diff
@@ -137,3 +137,6 @@ RTMR
 RTMRs
 precalculator
 initrd
+deduplicated
+unrepresentable
+serializable
```

Comment:
nit: please _keep-sorted_

Reply:
and please move the keep-sorted comment to the end of the list again.